README.md

```diff
@@ -85,6 +85,12 @@ The docs cover all aspects of using k6. Some highlights include:
 
 These links barely scratch the surface! If you're looking for conceptual information, you can read about [Test types](https://k6.io/docs/test-types/introduction/), [Test strategies](https://k6.io/docs/testing-guides/), or one of the many informative [Blog posts](https://k6.io/blog).
 
+## Roadmap
+
+Our team is dedicated to continuously improving and providing the best user experience possible. The [public roadmap](https://github.com/orgs/grafana/projects/443/views/1) covers user-oriented features, UX improvements and JavaScript support that our team will focus on. Remember that timeframes and priorities may shift, but we believe it's important to share our vision.
+
+We hope it provides a clear overview of our plans for future development. We welcome feedback, corrections, and suggestions to make it more comprehensive, accessible, and valuable for the community.
```

Comment:
Well put! :+1:

Reply:
I suggest having them in GitHub, which probably means adding one more thumb up to already open issues.

Reply:
```suggestion
Our team is dedicated to continuously improving and providing the best user experience possible. The [public roadmap](https://github.com/orgs/grafana/projects/443/views/1) covers user-oriented features, UX improvements and JavaScript support that our team will focus on. Remember that timeframes and priorities may shift, but we believe it's important to share our vision.

We hope it provides a clear overview of our plans for future development. We welcome feedback, corrections, and suggestions to make it more comprehensive, accessible, and valuable for the community. Adding one more :+1: reaction to a specific issue is an effective way of letting us know you want something prioritized higher.
```

What about it?

Reply:
https://github.com/orgs/community/discussions/17189

Reply:
A :+1: on an existing issue is helpful for us to prioritize the work, but it doesn't provide enough information for "feedback, corrections and suggestions". I think as long as we clarify with something like "We welcome feedback, corrections and suggestions via GitHub", it should be good enough. I doubt we would have a problem with people abusing this.

Reply:
```suggestion
Our team is dedicated to continuously improving and providing the best user experience possible. The [public roadmap](https://github.com/orgs/grafana/projects/443/views/1) covers user-oriented features, UX improvements and JavaScript support that our team will focus on. Remember that timeframes and priorities may shift, but we believe it's important to share our vision.

We hope it provides a clear overview of our plans for future development. We welcome feedback, corrections, and suggestions via GitHub to make it more comprehensive, accessible, and valuable for the community.
```